src/bench/bench.cpp

```diff
@@ -116,7 +116,7 @@ void benchmark::BenchRunner::RunAll(Printer& printer, uint64_t num_evals, double
         if (0 == num_iters) {
             num_iters = 1;
         }
-        State state(p.first, num_evals, num_iters, printer);
+        State state(p.first, num_evals, num_iters, scaling, printer);
```

Comment:
Why is this change required? Should be in a separate commit

Reply:
I need to disable the effect of the `scaling` setting, see above. The current implementation is a bit of a hack though; once refactored it can go in its own commit.